subprojects/gst-plugins-bad/sys/d3d11/gstd3d11d2d1.cpp

```diff
@@ -232,14 +232,15 @@ gst_d3d11_d2d1_transform(GstBaseTransform* trans, GstBuffer* inbuf,
             }
         }
 
-        ID2D1RenderTarget* render_target = gst_d3d11_memory_get_d2d1_render_target(mem, direct2DFactory);
+        ID2D1RenderTarget* render_target = nullptr;
+        gst_d3d11_memory_get_d2d1_render_target(nullptr, nullptr);
```

Comment:
this should be
```c
ID2D1RenderTarget* render_target = gst_d3d11_memory_get_d2d1_render_target(mem, direct2DFactory);
```
otherwise the code just won't work: it will actually just quit with error in the next line

Reply:
yes, I miss to recover the original change here because of the tests.... Thanks @aslobodeniuk !